src/stylesheets/base/_typography.scss

```diff
@@ -1,3 +1,19 @@
+//temp styling
+
 body {
   font-family: sans-serif;
 }
+
+img,
+svg{
+  height: 100px;
+}
+
+.emoji{
+  width: 40px;
+}
+
+
+section, header, nav, footer{
```

Comment:
when there are multiple selectors, we usually write them each on their own line:

```
section,
header,
nav,
footer {
```